src/test/java/org/jabref/model/entry/identifier/DOITest.java

```diff
@@ -316,4 +316,10 @@ public void rejectMissingDividerInShortDoi() {
     public void rejectNullDoiParameter() {
         assertThrows(NullPointerException.class, () -> new DOI(null));
     }
+
+    @Test
+    public void findDoiWithSpecialCharactersInText() {
```

Comment:
For the future: Integrate this in the exiting `@ParamterizedTest`. The method name is then the comment above the pair of `Arguments.of`. See Line 223.

Reply:
Had forgotten about this. Done in https://github.com/JabRef/jabref/pull/11603.